app/test/ColorPairTest.js

```diff
@@ -0,0 +1,44 @@
+import { ColorPair, getPairNumberFromColor, getColorFromPairNumber } from "../ColorPair.js";
+import { allColorPairs } from "../constants.js";
+
+// Test function
+function test() {
+
+    // Test all the pair num.
+    let allColorPairsIterator = 0;
+    for (let pairNum = 1; pairNum <= 25; pairNum++) {
+        let testPairColor = getColorFromPairNumber(pairNum);
+        console.log(`[In]Pair Number: ${pairNum}, [Out] Colors: ${JSON.stringify(testPairColor)}`);
+        console.assert(testPairColor.majorColor === allColorPairs.at(allColorPairsIterator).majorColor);
+        console.assert(testPairColor.minorColor === allColorPairs.at(allColorPairsIterator).minorColor);
+        allColorPairsIterator++;
+    }
+
+    // Test all the pair colors.
+    let pairNumIterator = 1;
+    for (let pairColorIterator = 0; pairColorIterator < 25; pairColorIterator++) {
+        let testPair = allColorPairs.at(pairColorIterator);
+        let pairNumber = getPairNumberFromColor(testPair);
+        console.log(`[In]Colors: ${JSON.stringify(testPair)}, [Out] Pair Number: ${pairNumber}`);
+        console.assert(pairNumber === pairNumIterator);
+        pairNumIterator++;
+    }
+
+    // Test invalid pair
```

Comment:
Nice tests for invalid inputs 👍 
The intention has come out clearly - we want the code to throw exceptions when inputs are invalid.
Only one comment: The tests will not fail, even if you remove the `throw` statements. Could you think of stronger tests, which will fail if the code does not throw?